apps/challenges/admin.py

```diff
@@ -11,6 +11,7 @@
     scale_workers,
     start_workers,
     stop_workers,
+    delete_evaluation_clusters,
```

Comment:
Please write this in alphabetical order.

Reply:
Updated